tests/integration/smoke_test.go

```diff
@@ -50,6 +50,10 @@ import (
 // - PUT/GET objects
 // ************************************************
 func TestSmokeSuite(t *testing.T) {
```

Comment:
With these changes, it might be good to add the `Ceph` keyword into Ceph's tests. e.g., `TestCephSmokeSuite`. That way if someone does `-test.list '*'` it's easy to see which tests are for which backend.